app/controllers/guides_controller.rb

```diff
@@ -27,6 +27,8 @@ def create
   def edit
     @guide = Guide.find(params[:id])
     @edition = @guide.latest_edition.unsaved_copy
+    @comments = @guide.latest_edition.comments.includes(:user)
+    @new_comment = @guide.latest_edition.comments.build
```

Comment:
last one: can we get rid of default_scope and order comments in here instead? It's subjective, but I really dislike default_scope, it has always made me regret using it in the past :/